cypress/component/unit_test_apollo_state_updates.spec.tsx

```diff
@@ -17,14 +21,21 @@ describe('Apollo state management', () => {
         );
         expect(dashboardItemsVar()).to.deep.equal([]);
     });
-    it('adds new item to state', () => {
-        // This has to change when we add more functionality to the Add component button.
+    it('state updates correctly', () => {
         mount(
-            <BrowserRouter>
-                <AddComponentButton />
-            </BrowserRouter>,
+            <MockedProvider mocks={mocks} cache={cache}>
+                <BrowserRouter>
+                    <Dashboard />
+                </BrowserRouter>
+            </MockedProvider>,
         );
-        cy.contains('Legg til nytt komponent').click();
-        expect(dashboardItemsVar()).to.not.equal([]);
+        const item = {
+            title: 'This is a custom title',
+            id: 'This is a custom ID',
+            size: DashboardItemSize.SMALL,
+            query: WEATHER_MET_API,
+        };
+        dashboardItemsVar([item]);
+        expect(dashboardItemsVar()).to.deep.include(item);
```

Comment:
This doesn't test our business logic, only if the library we're using works.

Reply:
👨‍💼